app/views/EventSummaryView.scala.html

```diff
@@ -25,7 +25,7 @@
 
 @(form: Form[_], waypoints: Waypoints)(implicit request: Request[_], messages: Messages)
 
-@layout(pageTitle = title(form, messages("eventSummary.title"))) {
+@layout(pageTitle = title(form, messages("eventSummary.title", "2023"))) {
```

Comment:
Should this be dynamic?

Reply:
Yes, for now as we don't have a page for selecting the year